catalogue/webapp/components/IIIFViewer/ImageViewer.tsx

```diff
@@ -83,6 +92,12 @@ const ImageViewer: FunctionComponent<ImageViewerProps> = ({
       })
       .join(',')
   );
+  const matching = rotatedImages.find(
+    canvas => queryParamToArrayIndex(canvas.canvasParam) === index
+  );
+
+  const rotation = matching ? matching.rotation : 0;
+  const lang = (work.languages.length === 1 && work?.languages[0]?.id) || '';
```

Comment:
Same as below, could the optional chaining be reviewed here? (work being optional is the second instance but not the first one and does the second instance need to be optional if we already know there's a work)